### PR TITLE
Alicia

### PR DESCRIPTION
**Pull request by aliciajpeel**
Number of people with cats and dogs

### CODE REVIEW

**Comment by ks-young** [comment]
excellent variable choice ;)

File: CollaborativeGit_SharedWorkbook.Rmd [hunk 1]
```diff
@@ -177,9 +177,9 @@ this will tell you if there is a correlation between ho often the participantd e
 
 
 
-```{r}
-
+```{r alicia workshop code}
 
+table(df$dog, df$cat)
```
Comment (klpurves) [inline CollaborativeGit_SharedWorkbook.Rmd:182]: I love  what you did here, super interesting.

File: File_info.R [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Assign your downloaded datapath to the variable below
 
-datapath <- "path/to/data"
-  
\ No newline at end of file
+datapath <- "/Users/aliciapeel/Desktop/nutrition-studies/raw_anonymized_data.csv"
```
Comment (klpurves) [inline File_info.R:3]: Concerned for data safety here - could you check the path maybe?

**Review by klpurves — APPROVED** [review]
great job

**Comment by klpurves** [comment]
> excellent variable choice ;)

Totally agree

**Pull request merged**